### DISCUSSION TITLE
Custom /.well-known/

### DISCUSSION THREAD

**Discussion by nagoorkani** [panva/openid-client#593] [Ideas]
Hello,

We are using nextjs13 for one our app. We've discovered an issue within the auth flow due to the our custom `well-known` config.  Making a small tweak to the existing code solves our problem `if (parsed.pathname.includes('/.well-known'))` in `issuer.js`

Please let me know.

**Comment by panva** [comment]
Custom well known is already supported so long as `/.well-known/` is in the URL passed to discover. Anything beyond that is out of scope. You can always fetch the response of your well known yourself and construct the Issuer instance off of the returned json.

**Reply by nagoorkani** [reply]
Thanks for the quick response 👍 we've customized as `/.well-knownxyz/` which is failing in the conditions. So, removing the end `/` will help fix our problems.
